bpf/lib/process.h

```diff
@@ -326,6 +328,10 @@ struct binary {
 	// NB: everything after and including ->mb_bitset will not be zeroed on a new exec. See
 	// binary_reset().
 	mbset_t mb_bitset;
+	// length of the args stored in args
+	__s64 args_length;
+	// args for the binary
+	char args[MAXARGLENGTH];
```

Comment:
Can you please place these above `->mb_bitset`? See comment above. The idea is that we zero out the paths but not `->mb_bitset` on a new exec.

Reply:
I can do we need to zero out the paths though? Here I just null terminated it.